client/src/ctx.rs

```diff
@@ -242,7 +269,10 @@ impl SessionContext {
 
         match resp.status() {
             OK => {
-                let body = resp.bytes().await?;
+                let mut body = resp.bytes().await?;
+                if let Some(encoding) = self.session_config.accept_encoding {
```

Comment:
Sometimes the client user set multiple accept_encodings but crate `fly_accept_encoding` only gives you the first codec, when the server responses another encoding can it handle properly?